jekyll/_cci2/certificates.adoc

```diff
@@ -193,8 +193,26 @@ Once you have a valid certificate and key file in `.pem` format, you must upload
 
 . Click "Save" at the bottom of the settings page and restart when prompted
 
-More information is available https://letsencrypt.readthedocs.io/en/latest/using.html#manual[here].
-
 Ensure the hostname is properly configured from the Management Console (`<circleci-hostname>.com:8800`) **and** that the hostname used matches the DNS records associated with the TLS certificates.
 
 Make sure the Auth Callback URL in GitHub/GHE matches the domain name pointing to the Services machine, including the protocol used, for example `**https**://info-tech.io/`.
+
+=== Adding the certificate to Replicated
+
+ Replicated Management Console(`<circleci-hostname>.com:8800`) runs on different web server, so you need to apply the above certificate for Replicated as well.
+
+. To do so, navigate to `https://hostname:8800/console/settings#tls-key-cert`
+
+. Upload your newly generated certificate and key
+
+. Click "Save" at the bottom of the page and restart the Replicated UI when prompted
+
+Even if you can't access the Replicated Page with the hostname, you still can access by IP address.
+
+Also, as alternative option, you can upload the certificate and key by following command on the ServiceBox.
```

Comment:
```suggestion
Alternatively, you can upload the certificate and key by running the following command on the Services machine, substituting your hostname, path to key and path to certificate:
```

Reply:
Is <domain> here the hostname?

Reply:
Yes, in this context, I should use `<circleci-hostname>`. So, I'll update the following script.